src/eventBus.js

```diff
@@ -0,0 +1,7 @@
+const EventEmitter = require('events');
+
+class Events extends EventEmitter {}
+
+const bus = new Events();
+
+module.exports = bus;
```

Comment:
Do we really need a new file for this?

Reply:
Yes, creating a new file means all other files (even the user's code) can use the same eventEmitter class which is an absolute requirement to make this pattern work.